standalone/src/DelphesMain.h

```diff
@@ -3,7 +3,11 @@
 #include "k4SimDelphes/DelphesEDM4HepOutputConfiguration.h"
 
 #include "podio/Frame.h"
+#if PODIO_VERSION_MAJOR > 0 || (PODIO_VERSION_MAJOR == 0 && PODIO_VERSION_MINOR >= 99)
+#include "podio/ROOTReader.h"
+#else
 #include "podio/ROOTFrameWriter.h"
+#endif
```

Comment:
```suggestion
#include "podio/ROOTFrameWriter.h"
namespace podio {
using ROOTWriter = podio::ROOTFrameWriter;
}
#endif
```

Save us another `#if` below.